### ISSUE TITLE
Include CHANGELOG.md in the documentation

### ISSUE THREAD

**Issue by stevepiercy** [plone/volto#3992]
**Is your feature request related to a problem? Please describe.**

Many projects include their change log somewhere in their docs. Volto does not. Its [upgrade guide](https://6.dev-docs.plone.org/volto/upgrade-guide/index.html) is narrative text, and not just a list of changes.

Example projects:
- [Plone](https://plone.org/download/releases), but you need to click each release to get its release notes. Maybe this is something that @mauritsvanrees and @esteele should consider for overall Plone 6 Documentation?
- [Django Release Notes](https://docs.djangoproject.com/en/4.1/releases/)
- [Pyramid Change History](https://docs.pylonsproject.org/projects/pyramid/en/latest/changes.html)

**Describe the solution you'd like**

Create a new page, "Volto Release Notes", at `release-notes/index.md`. This page would consist primarily of a MyST include of CHANGELOG.md:

```md
{include} ../../../CHANGELOG.md
```

See also https://myst-parser.readthedocs.io/en/latest/faq/index.html#include-a-file-from-outside-the-docs-folder-like-readme-md

Opinions @plone/volto-team?

**Comment by davisagli** [comment]
+1 from me, since it would presumably make it easier to find via search

**Comment by stevepiercy** [comment]
It also will validate links and syntax errors. I just found one while testing. There needs to be a conditional include depending on whether only Volto docs are being built or the entire Plone 6 docs, but otherwise this works pretty well.

In my testing, I realized that there would be two titles. Should we use the title "Volto Release Notes" or "Change Log"?

![Screen Shot 2022-11-27 at 9 40 33 PM](https://user-images.githubusercontent.com/102112/204202361-d1835c51-4d9d-4474-ae3a-fb9cde4f8f8c.png)
